root/etc/e-smith/templates/etc/nethserver/cockpit.allow/10base

```diff
@@ -0,0 +1,19 @@
+
+#
+# Allow groups to use cockpit 
+#
+root
+domain admins
```

Comment:
Here we'd expand the `admins/group` prop too...
```suggestion
{ $admins{'group'} }
```

Reply:
done